pkg/crdconversion/crdconversion.go

```diff
@@ -75,15 +79,16 @@ func (crdWh *crdConversionWebhook) run(stop <-chan struct{}) {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
-	mux := http.NewServeMux()
+	wg := new(sync.WaitGroup)
```

Comment:
Not sure if you were planning to use it to wait post-shutdowns, to ensure the routines finished. Otherwise this looks unused so far.